### ISSUE TITLE
Idea - Store GPS position and navigation feature

### ISSUE THREAD

**Issue by tiuksferve** [ezod/flipperzero-gps#6]
Store GPS positions can be useful to implement simple navigation feature like mini GPSs

https://www.joom.com/pt-br/products/5d8eea051436d40101e68228

This feature store a position (like home) and it can plot an arrow pointing to position's direction and the distance.
Analog Clock can be a good app too check how to plot the arrow.

**Comment by sambyte61** [comment]
Being able to visually describe the direction between your current location and a home location would require a digital compass?  You could calculate the distance between your current location and your home location and determine an azimuth from that, but there'd be no way to display that visually to the user as your holding it without a compass.

**Comment by magopeter** [comment]
> Being able to visually describe the direction between your current location and a home location would require a digital compass? You could calculate the distance between your current location and your home location and determine an azimuth from that, but there'd be no way to display that visually to the user as your holding it without a compass.

That is true if you are stationary, but it can tell you a direction if you are moving ;)
Old school GPS never knew the direction unless you were on the move :)
